### DISCUSSION TITLE
2.3.1 Released!

### DISCUSSION THREAD

**Discussion by cmmarslender** [Chia-Network/chia-blockchain#18097] [Announcements]
🌱 Version 2.3.1 of the Chia reference client is now available for download! Upgrading to this release is optional. It includes a single update - the Chia reference wallet now contains the Warp.green assets by default.

[Download 2.3.1](https://www.chia.net/downloads/)

Read our (short) blog on the update here:

https://www.chia.net/2024/05/01/version-2-3-1-release/

### Added
* Added `warp.green` CATs (`wUSDC.b`, `wmilliETH.b`, `wUSDC`, `wmilliETH`, `wUSDT`) to the known CAT list

<hr /><em>This discussion was created from the release <a href='https://github.com/Chia-Network/chia-blockchain/releases/tag/2.3.1'>2.3.1</a>.</em>

**Comment by svenove7** [comment]
2.01 is working and 2.0.0 and 1.8.1 but 2.1.0  2.1.1 and 2.3.1 doesn't work. They says they are starting the service of the full node and I have waited the whole night so it is never happening. I tried going back and forth in the versions and it seems the new ones aren't working.
I am running win10 could upgrading to win11 help ?
I have seen that others have had the same problem in the past with other versions and I think I have had this problem or similar too in much earlier versions.

**Reply by BrandtH22** [reply]
Hey @svenove7 , I created a support ticket for you as #18178 and responded there with some solutions to try.

as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia).
